src/visit_vtk/lightweight/vtkUnstructuredGridFacelistFilter.h

```diff
@@ -33,14 +33,14 @@ class VISIT_VTK_LIGHT_API vtkUnstructuredGridFacelistFilter :
     public vtkPolyDataAlgorithm
 {
   public:
-    vtkTypeMacro(vtkUnstructuredGridFacelistFilter, vtkPolyDataAlgorithm);
+    vtkTypeMacro(vtkUnstructuredGridFacelistFilter, vtkPolyDataAlgorithm)
     void PrintSelf(ostream& os, vtkIndent indent) override;
 
     static vtkUnstructuredGridFacelistFilter *New();
 
   protected:
-    vtkUnstructuredGridFacelistFilter() {;};
-    ~vtkUnstructuredGridFacelistFilter() {;};
+    vtkUnstructuredGridFacelistFilter() {;}
```

Comment:
Same question here. Do we really need these two lines or does removing them and just having default constructor and destrutor do the right thing?

Reply:
I don't know. Since the warnings are fixed can they be left as-is?